cpplite/adls/include/adls_client.h

```diff
@@ -315,6 +315,11 @@ namespace azure { namespace storage_adls {
             return m_exception_enabled;
         }
 
+        AZURE_STORAGE_ADLS_API std::shared_ptr<azure::storage_lite::blob_client> get_blob_client()
+        {
```

Comment:
remove

Reply:
This method we need to add because there is no way to get the implicit blob client object from within the adls object as of now.